app/services/case_filter/open_case_status_filter.rb

```diff
@@ -26,11 +26,21 @@ def available_choices
           end
         end
       end
+
+      move_state_filter_option_to_end(state_choices, "rejected")
```

Comment:
Just thinking about this again. Do the options need to be in a particular order? If so, would it be the order where they are originally defined, in which case can you not change that order instead of doing it here?

Reply:
Hi the ticket only asks for 'rejected' to be at the end. The order is defined within the workflow file 'config/state_machine/configs/00_moj/40_offender_sar/10_offender_sar_preamble.yml', with `rejected` near the bottom of `permitted states` (`closed` is not shown). 

I did consider your suggestion however it seems that `{ filter_open_case_status: state_choices }` returns most of the status options. Then when `available_choices` is looped through within the view `app/views/cases/search_filters/_filter_multiple_choices.slim`, more options are added which seem to relate to SAR complaints only, so `rejected' is no longer at the bottom. `filter.available_choices[filter_attribute]` within the view adds on the extra filter options. I'm not sure how to resolve this in a different way.

Reply:
I think `describe ".process_params!" do` within `open_case_status_filter_spec.rb` tests for list order, however I'm not sure how to do a test for the full view to ensure that 'rejected' appears after the SAR complaint status filters as well.